### ISSUE TITLE
Request to merge dotnet test and dotnet vstest into single command

### ISSUE THREAD

**Issue by ahaleiii** [microsoft/vstest#1453]
Currently there are two different ways to run tests through dotnet cli:
$ dotnet test
$ dotnet vstest

According to @Faizan2304 in #637, these are both wrappers for the vstest.console.dll already and should be no real behavior differences between them. ([source](https://github.com/Microsoft/vstest/issues/637#issuecomment-291762855)) As a CI/CD scripter, I want the ability to run tests with the dotnet cli without having to remember the distinction between the two, so that I can go faster and focus on more important details.

My acceptance criteria would be along the lines of:
- The "dotnet test" command can be used in place of the "dotnet vstest" command.
- Using "dotnet test" and targeting a dll, the inclusion or exclusion of the "--no-build" argument will have no impact on the behavior of execution.

Stretch goals:
- The "dotnet test" command can accept a list of dlls
- Using "dotnet test" and targeting a list of dlls, there is an argument for "--parallel" which will run the tests from the dlls in parallel.

Background:
I am looking into implementing new portions of my organization's CI/CD pipeline for our .NET Core applications. Some of those activities involve running unit and integration tests. I would like the ability to pass in the filename of either a .csproj or a .dll without having to change any other syntax of the dotnet test command.

**Comment by cltshivash** [comment]
@ahaleiii Thanks for the suggestion. We don't have any immediate plans for the above.

**Comment by singhsarab** [comment]
I have created RFC for above ask and mentioned the issues that need to be addressed.
RFC: https://github.com/microsoft/vstest-docs/pull/196

**Comment by singhsarab** [comment]
This is available in https://dotnetcli.blob.core.windows.net/dotnet/Sdk/release/3.1.1xx/dotnet-sdk-latest-win-x64.exe
